test/e2e/feature/feature.go

```diff
@@ -468,12 +468,6 @@ var (
 	// TODO: document the feature (owning SIG, when to use this feature for a test)
 	Upgrade = framework.WithFeature(framework.ValidFeatures.Add("Upgrade"))
 
-	// Owned by SIG Node
-	// Can be used when the UserNamespacesPodSecurityStandards kubelet feature
-	// gate is enabled to relax the application of Pod Security Standards in a
-	// controlled way.
-	UserNamespacesPodSecurityStandards = framework.WithFeature(framework.ValidFeatures.Add("UserNamespacesPodSecurityStandards"))
```

Comment:
I was wondering myself if we should simply remove it or lock to default like here: https://github.com/kubernetes/kubernetes/pull/134691

In both cases the FG was just added "in case it will break something big".

We may need some common guidance on this, I do not care either way

Reply:
I do think in this case the point of putting it here was to never let it move to beta. Open to other guidance

Reply:
I meant moving it to lock to default or something.

Reply:
we have three patterns now:

- alpha FG: this PR
- straight to beta: https://github.com/kubernetes/kubernetes/pull/134691
- Deprecated FG:  https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/4781-kublet-restart-pod-status/README.md#design-details

and I didn't find a good example on the right way to remove these FGs